dev-util/muon/muon-0_pre20220714.ebuild

```diff
@@ -29,12 +30,17 @@ DEPEND="
 	curl? ( net-misc/curl )
 	archive? ( app-arch/libarchive:= )
 	libpkgconf? ( dev-util/pkgconf:= )
-	man? ( app-text/scdoc )
+	man? (
+		app-text/scdoc
+		dev-python/pyyaml
+	)
 "
 RDEPEND="${DEPEND}"
+BDEPEND="${PYTHON_DEPS}"
 
 src_prepare() {
 	default
+	python_setup
```

Comment:
Shouldn't need this bc exports pkg_setup but you do need to define python_check_deps (see python guide)